SharedUtils/build.gradle

```diff
@@ -39,10 +39,10 @@ dependencies {
     testImplementation 'org.mockito:mockito-core:3.0.0'
     testImplementation 'org.robolectric:robolectric:4.1'
 
-    testImplementation 'org.powermock:powermock-module-junit4:2.0.7'
-    testImplementation 'org.powermock:powermock-module-junit4-rule:2.0.7'
-    testImplementation 'org.powermock:powermock-api-mockito2:2.0.7'
-    testImplementation 'org.powermock:powermock-classloading-xstream:2.0.7'
+    testImplementation 'org.powermock:powermock-module-junit4:2.0.9'
+    testImplementation 'org.powermock:powermock-module-junit4-rule:2.0.9'
+    testImplementation 'org.powermock:powermock-api-mockito2:2.0.9'
+    testImplementation 'org.powermock:powermock-classloading-xstream:2.0.9'
```

Comment:
Test weren't passing because of Powermock / Mockito. Bumping this version helped. We should remove Powermock one day.

Reply:
![Do it](https://media2.giphy.com/media/3o84sw9CmwYpAnRRni/giphy.gif)